tests/integration/test_api_gateway.py

```diff
@@ -179,20 +180,49 @@ def test_api_gateway_lambda_proxy_integration():
     result = connect_api_gateway_to_http_with_lambda_proxy('test_gateway2', target_uri,
         path=API_PATH_LAMBDA_PROXY_BACKEND)
 
+    api_id = result['id']
+    path_map = get_rest_api_paths(api_id)
+    _, resource = get_resource_for_path('/lambda/foo1', path_map)
+
     # make test request to gateway and check response
     path = API_PATH_LAMBDA_PROXY_BACKEND.replace('{test_param1}', 'foo1')
     path = path + '?foo=foo&bar=bar&bar=baz'
 
-    url = INBOUND_GATEWAY_URL_PATTERN.format(api_id=result['id'], stage_name=TEST_STAGE_NAME, path=path)
+    url = INBOUND_GATEWAY_URL_PATTERN.format(api_id=api_id, stage_name=TEST_STAGE_NAME, path=path)
     data = {'return_status_code': 203, 'return_headers': {'foo': 'bar123'}}
-    result = requests.post(url, data=json.dumps(data))
+    result = requests.post(url, data=json.dumps(data), headers={'User-Agent': 'python-requests/testing'})
     assert result.status_code == 203
     assert result.headers.get('foo') == 'bar123'
     parsed_body = json.loads(to_str(result.content))
     assert parsed_body.get('return_status_code') == 203
     assert parsed_body.get('return_headers') == {'foo': 'bar123'}
     assert parsed_body.get('pathParameters') == {'test_param1': 'foo1'}
     assert parsed_body.get('queryStringParameters') == {'foo': 'foo', 'bar': ['bar', 'baz']}
+
+    request_context = parsed_body.get('requestContext')
+    source_ip = request_context['identity'].pop('sourceIp')
+
+    assert re.match(re.compile(r'^\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}$'), source_ip)
+    assert request_context == {
+        'path': '/lambda/foo1',
+        'accountId': '000000000000',
```

Comment:
Instead of hardcoding the string, let's better use `TEST_AWS_ACCOUNT_ID` from `constants.py`